os/arch/arm/src/amebalite/amebalite_serial.c

```diff
@@ -558,7 +558,7 @@ static int rtl8720e_log_uart_irq(void *Data)
 
 static int rtl8720e_log_up_attach(struct uart_dev_s *dev)
 {
-	InterruptRegister((IRQ_FUN)rtl8720e_log_uart_irq, RTL8720E_UART_LOG_IRQ-16, NULL, 4);
+	InterruptRegister((IRQ_FUN)rtl8720e_log_uart_irq, RTL8720E_UART_LOG_IRQ-16, NULL, 0);
```

Comment:
@fengyunlai COuld you please see why UART Rx has an issue with LOGUART (4) ?

Reply:
On latest loadable_apps configuration*

Reply:
Hi Thapa, we have similar issue for our IPC irq, which we are debugging right now. (if we set a higher priority to IPC irq like you did for loguart irq, the issue is temporarily fixed also) Actually all these issues happen after we upgrade docker from 1.5.5 to 1.6.0. I will send an email to Sagam later today. Thank you.